main.py

```diff
@@ -148,14 +148,41 @@ def FillHandWithTiles(TileQueue, PlayerTiles, MaxHandSize):
 
 
 def GetScoreForWord(Word, TileDictionary):
-    Score = 0
-    for Count in range(len(Word)):
-        Score += TileDictionary[Word[Count]]
-    if len(Word) > 7:
-        Score += 20
-    elif len(Word) > 5:
-        Score += 5
-    return Score
+  done1 = 0
+  done2 = 0
+  Score = 0
+  NewWord = [] 
+  for Count in range (len(Word)):
+    Score += TileDictionary[Word[Count]]
+  if len(Word) > 7:
+    Score += 20
+  elif len(Word) > 5:
+    Score += 5
+  for Letters in range(len(Word)):
+      NewWord.append(Word[Letters])
```

Comment:
replace with `NewWord = Word`

Reply:
I can't do that because Word is the name of the variable used by the user input